### DISCUSSION TITLE
Handle hexadecimal numeric literals in calculator

### DISCUSSION THREAD

**Discussion by PAI5REECHO** [Flow-Launcher/Flow.Launcher#1171] [Ideas]
**Is your feature request related to a problem? Please describe.**
The calculator doesn't parse hexadecimal numeric literals at all

**Describe the solution you'd like**

Add hexadecimal numeric literals to calculator plugin

**Comment by taooceros** [comment]
Move to discussion. I will give it a check if it is not hard to implement later

**Comment by CharredRich** [comment]
@taooceros any chance this will be implemented?  It would be real nice to see 2,123,456 instead of the hard to read 2123456. - Thank you.
